src/rgw/rgw_rados.cc

```diff
@@ -10028,10 +10110,11 @@ RGWRados *RGWStoreManager::init_raw_storage_provider(CephContext *cct)
   store = new RGWRados;
 
   store->set_context(cct);
+  store->init_ctxpool();
```

Comment:
`init_ctxpool()` isn't being called in the `init_storage_provider()` case

Reply:
cherry-picked